docs/pages/identity-governance/access-lists/terraform.mdx

```diff
@@ -1,348 +0,0 @@
----
-title: Managing Access Lists with Terraform
-description: Learn how to manage Access Lists and their members with Terraform.
-tags:
- - infrastructure-as-code
- - how-to
- - identity-governance
- - privileged-access
----
-
-You can manage Access List and their members using Terraform. This guide will help you:
-
-- Understand how to manage Access Lists with Terraform
-- Understand the difference between Access Lists managed by Teleport vs Terraform
-- Understand how to manage Access List members with Terraform
-
-
-## How it works
-
-We will create a simple organization structure represented with Access Lists using
-Terraform. It will consist of:
-
-- Access List representing a team
-- Access List representing access to a resource (staging database)
-- assigning the team access to the resource
-- (optionally) assigning Okta and/or Entra ID group access to the resource
-
-Access List management with Terraform is done using 2 Terraform resources:
-`teleport_access_list` and `teleport_access_list_member`.
-
-There are some nuances to members management. This is where concept of *Access List type* comes
-into play. There are 2 key Access List types: *default* and *static*.
-
-### Access List of default type
-
-The default Access List has the `.spec.type` field unset, or set to null or an empty string. Those are
-regular Access Lists, like those created in the web UI. And as a consequence:
-
-- They require auditing. Even though `.spec.audit` is not required, to be specified in the
-  Terraform resource, the default value will be set and those lists have to be periodically
-  reviewed in the web UI.
-- Their members can be only managed with with the web UI and `tctl`. The source of truth
-  for these lists is Teleport so you cannot manage their members with Terraform.
-
-
-### Access List of static type
-
-Access List of type static, have `.spec.type` set to "static". They differ from the default Access Lists:
-
-- They don't support audit. `.spec.audit` field can be set, but it's ignored by Teleport.
-- Their members can be managed by Terraform.
-
-For static Access Lists, the source of truth for the membership is external (provisioned by
-Terraform) so audit is not supported. The members should be reviewed at source, which are the
-Terraform data sources or manifests and this process has to be external to Teleport.
```

Comment:
This explanation is important but is missing from the new page https://github.com/gravitational/teleport/pull/60844/files#diff-7a487cfdfa16c5c6ca5d0653373db92bfb2eb73edc840f109e7295c9a6efb1b6